src/main/scala/com/atomist/rug/test/gherkin/AbstractExecutableFeature.scala

```diff
@@ -45,6 +45,8 @@ abstract class AbstractExecutableFeature[W <: ScenarioWorld](
             runGiven(world, step)
           case "When " if !world.aborted =>
             runWhen(world, step)
+          case "Then " if step.getText == "the scenario aborted" =>
```

Comment:
I agree that would be a good enhancement. I didn't really touch any of that code here though. We'd have to look at how to best do it.